File: API/Entities/Other.cs
namespace API.Entities;

public class Other
{
    public string Id { get; set; } = Guid.NewGuid().ToString();
    public required string Name { get; set; }
    public required int Price { get; set; }
    public required int Quantity { get; set; }
    public bool IsArchived { get; set; }
    public DateTime? ArchivedAt { get; set; }
}

File: CLIENT/src/types/Other.ts
export interface Other {
    id: string;
    name: string;
    price: number;
    quantity: number;
    isArchived: boolean,
    archivedAt: Date,
    images: OtherImage[];
}

export interface OtherImage {
    id: number;
    url: string;
    publicId: string;
    otherId: string;
    isMain: boolean;
}
